llama_index/postprocessor/sbert_rerank.py

```diff
@@ -18,6 +18,7 @@ def __init__(
         self,
         top_n: int = 2,
         model: str = "cross-encoder/stsb-distilroberta-base",
+        device: str = "cuda",
```

Comment:
should by default `cpu`?

Reply:
Probably, users want to use `cuda`, if it's available. So I would leave it like that tbh.

Reply:
can you do like this 
```
device = "cuda" if torch.cuda.is_available() else "cpu"
```